user-guide/content/docs/installation.md

```diff
@@ -0,0 +1,97 @@
+---
+weight: 3
+title: Installation Instructions
+---
+
+## Using Docker (recommended)
+
+Docker images are continuously built and deployed on [dockerhub](https://hub.docker.com/u/inputoutput) under specific tags. Using docker provides **the fastest** and **easiest** user experience for setting up the Cardano stack. You should prefer this solution over building from sources unless you have really good reasons not to. The following images are available for each component of the Adrestia architecture:
+
+| Repository                                                           | Tags                          | Documentation               |
+| :---                                                                 | :---:                         | :---:                       |
+| [inputoutput/cardano-node][inputoutput-cardano-node]                 | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-node]    |
+| [inputoutput/cardano-db-sync][inputoutput-cardano-db-sync]           | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-db-sync] |
+| [inputoutput/cardano-graphql][inputoutput-cardano-graphql]           | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-graphql] |
+| [inputoutput/cardano-explorer-api][inputoutput-cardano-explorer-api] | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-rest]    |
```

Comment:
Documentation for cardano-node and cardano-db-sync link to `docker.nix`... I fear of those links a bit... (cause it is part of live, moving code)
`cardano-graphql` links to https://github.com/input-output-hk/cardano-graphql/wiki/Docker which does not exist atm, oh well.

Reply:
Yeah, the graphql one still lives in a branch... And, there's no Docker documentation for cardano-node and cardano-db-sync apart from these bits in the nix file written by rhys... I've asked the teams already to write some but... * sounds of silence *